dist/ng-showdown.js

```diff
@@ -100,36 +119,62 @@
       }
 
       /**
-       * AngularJS Directive to Md to HTML transformation
+       * @deprecated
+       * Legacy AngularJS Directive to Md to HTML transformation
        *
        * Usage example:
        * <div sd-model-to-html="markdownText" ></div>
        *
        * @param {showdown.Converter} $showdown
+       * @param {$sanitize} $sanitize
        * @param {$sce} $sce
        * @returns {*}
        */
-      function markdownToHtmlDirective($showdown, $sce) {
+      function sdModelToHtmlDirective($showdown, $sanitize, $sce) {
         return {
           restrict: 'A',
-          link: link,
+          link: getLinkFn($showdown, $sanitize, $sce),
           scope: {
             model: '=sdModelToHtml'
           }
         };
+      }
+
+      /**
+       * AngularJS Directive to Md to HTML transformation
+       *
+       * Usage example:
+       * <div markdown-to-html="markdownText" ></div>
+       *
+       * @param {showdown.Converter} $showdown
+       * @param {$sanitize} $sanitize
+       * @param {$sce} $sce
+       * @returns {*}
+       */
+      function markdownToHtmlDirective($showdown, $sanitize, $sce) {
+        return {
+          restrict: 'A',
+          link: getLinkFn($showdown, $sanitize, $sce),
+          scope: {
+            model: '=markdownToHtml'
+          }
+        };
+      }
 
-        function link(scope, element) {
+      function getLinkFn($showdown, $sanitize, $sce) {
+        return function (scope, element) {
           scope.$watch('model', function (newValue) {
-            var val;
+            var val,
+                showdownHTML;
             if (typeof newValue === 'string') {
-              var showdownHTML = $showdown.makeHtml(newValue);
-              val = $sce.trustAsHtml(showdownHTML);
+              showdownHTML = $showdown.makeHtml(newValue);
+              val = ($showdown.getOption('sanitize')) ? $sanitize(showdownHTML) : $sce.trustAsHtml(showdownHTML);
```

Comment:
I haven't run the code again, but in the non-sanitize case looks like ut is **still** broken. The old code is unchanged, but using $sce.trustAsHtml doesn't make any sense in this case. It would be used to pass the HTML as trusted to other directives, but that is not the case here. It is directly used in element.html(). IMHO there is no need for $sce at all here.